redis/src/main/scala/zio/redis/options/Scripting.scala

```diff
@@ -0,0 +1,12 @@
+package zio.redis.options
+
+import zio.Chunk
+import zio.redis.RedisEncoder
+
+trait Scripting {
+
+  sealed case class Script[K, A](lua: String, keys: Seq[K], args: Seq[A]) {
```

Comment:
As mentioned in my previous comments, this isn't an option, and it isn't even necessary anymore.

Reply:
Script has been deleted. IMHO this class showed relation between keys, args and lua script, that's why i wrote it)